runelite-client/src/main/java/net/runelite/client/plugins/suppliestracker/ChargesBox.java

```diff
@@ -0,0 +1,68 @@
+package net.runelite.client.plugins.suppliestracker;
+
+import net.runelite.client.game.ItemManager;
+import net.runelite.client.util.QuantityFormatter;
+import static net.runelite.api.ItemID.*;
+
+class ChargesBox extends Box
```

Comment:
To make this class easier to read, you should use indentations and formatting like this, https://gist.github.com/Ganom/098da195cb30b009d167312c6243f370

Reply:
Added that style to all boxes, changed if statements with multiple conditions to that format, and added extra comments/ separated declarations.